docs/make.jl

```diff
@@ -47,6 +47,7 @@ makedocs(;
         "Home" => "index.md",
         "Getting Started" => "getting_started.md",
         "How It Works" => "how_it_works.md",
+        "Model Providers and Supported Functions" => "model_providers_and_supported_functions.md",
```

Comment:
Could we please make it shorter? It will stretch the sidebar width.

Maybe "Coverage of Model Providers"? Any other ideas?

Reply:
I like both! Sizewise it's not that different, so pick your favourite.

Or it could be "Provide Coverage by Function" (meant as ai* functions), but I'll leave that to you.

Or "Model Provide Coverage" (to avoid the "of")

Reply:
I think your initial suggestion, "Coverage of Model Providers", would be the easiest to understand. So maybe we should go with that.